opacclient/opacapp/src/main/res/values-de/strings.xml

```diff
@@ -288,4 +289,15 @@
     <string name="starred_removed_undo">Rückgängig</string>
     <string name="print">Drucken</string>
     <string name="print_wait">Bitte warten, bis die Details geladen wurden!</string>
+    <string name="preference_category_library_config">Bibliotheksdaten</string>
+    <string name="preference_update_library_config">Konfiguration für Bibliotheken aktualisieren</string>
+    <string name="preference_reset_library_config">Konfiguration zurücksetzen</string>
```

Comment:
I believe "Konfiguration" is misleading, as it sounds like the user could change it. Maybe just "Bibliotheksdaten aktualisieren/zurücksetzen"?